mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx

```diff
@@ -491,7 +492,9 @@ export function CompositeSwapScreen ({ route }: Props): JSX.Element {
               fee={fee}
               isConversionRequired={isConversionRequired}
               slippage={slippage}
-              onSetSlippage={(val: BigNumber) => setSlippage(val)}
+              onSetSlippage={(val: BigNumber) => {
+                void setSlippage(val)
```

Comment:
any reason to add `void` here?

Reply:
setSlippage is a Promise, so it must be marked with a void